NEWS

```diff
@@ -113,7 +113,10 @@ New Functionality
   environment variable configures the addition.
 
 - SMB2 packets containing multiple PDUs now correctly parse all of the headers,
-  instead of just the first one and ignoring the rest.
+  instead of just the first one and ignoring the rest. This has the potential to
+  cause decreased performance on SMB2-heavy networks. Since the remaining PDUs
```

Comment:
I think I'd reword as "This may increase worker CPU usage on SMB2-heavy networks." and probably skip the parsing and more events part. Seems a bit redundant or too detailed.